page-components/form-fields.md

```diff
@@ -63,9 +63,9 @@ For screen readers, make sure the tab focus order is correct. Consider the order
 
 #### Default
 
-* Border: 1 px #75787b (Grey)
-* Height: 27 px
-* Padding: 10px 3px
+* Border: 1 px #919395 (Grey 60)
+* Height: 35 px (includes border, padding, and text height)
```

Comment:
I think he's just talking about the space in "35 px" and "1 px", as opposed to "35px" or "1px" with no space.